content/reference/user/simulations/index.md

```diff
@@ -75,8 +75,16 @@ In order to create a simulation click on the "Create" button in the simulations
 
 In this step, you'll configure the locations used for the Gatling Enterprise load generators.
 
-Gatling Enterprise locations are available in the following regions:
+You can either use the public locations provided by Gatling Enterprise Cloud, or use your own.
 
+{{< alert info >}}
+It is not possible to mix public and private locations in the same simulation.
```

Comment:
```suggestion
It is not currently possible to mix public and private locations in the same simulation.
```
Foreshadowing. (Or maybe not since we don't know when we'll get to it?)